addOns/encoder/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Maintenance changes.
 
+### Added
+- A context menu active on the output text areas to allow to replace the input area text with the relevant output.
```

Comment:
`input area text` does not sound right, is the order correct?

Reply:
It’s “right”, but you’re right that it is awkward, I’ll try to rewrite it.